packages/cursorless-engine/src/processTargets/modifiers/EveryScopeStage.ts

```diff
@@ -121,3 +120,19 @@ export class EveryScopeStage implements ModifierStage {
     return iterationScopeTarget.contentRange;
   }
 }
+
```

Comment:
not related to this PR, but no reason to have this small function in its own file when it's now only used here